app/src/main/res/layout/settings_content.xml

```diff
@@ -8,8 +8,7 @@
     android:background="@color/black"
     app:layout_behavior="@string/appbar_scrolling_view_behavior"
     tools:showIn="@layout/app_bar_layout"
-
-    >
+    android:background="@color/white">
 
   <FrameLayout
       android:id="@+id/settings_framelayout"
```

Comment:
Use a light background color other than white because the switch is not visible properly when it is off in white background.
![image](https://user-images.githubusercontent.com/34706326/60408837-29edb500-9bde-11e9-99b5-43c6dbaf1018.png)